.github/settings.yml

```diff
@@ -0,0 +1,22 @@
+repository:
```

Comment:
It would be good if we can also set the GitHub Pages settings in accordance with our pkgdown workflow. Let's keep an eye on https://github.com/repository-settings/app/issues/609.